### ISSUE TITLE
Update john-travoltage images

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/john-travoltage#121]
Request was made on 10/24/16 to update some images in the sim.  New assets in 'voltage black.ai' of https://drive.google.com/drive/u/0/folders/0B-YvjkIIcS_aRVFvZ2NDYlZ6Qkk.

Should be done before the end of the week.

**Comment by jessegreenberg** [comment]
Here is  a (very rough) version with the new images.  A few things still need to be tweaked so that things line up correctly.  Also, I will need to change the model so that charges align with the new body.

http://www.colorado.edu/physics/phet/dev/html/john-travoltage/1.3.0-dev.8/john-travoltage_en.html

Before doing this, it seems there are a couple issues with the new assets:

 - The brighter colors on the back leg makes it obvious that the legs 'cross over' during the drag.  Is this OK?
 - The back leg was designed to be in the back, behind the body.  This makes the shading look incorrect, and there is a portion of the back leg that was meant to be hidden behind the front leg.
 - The leg image wasn't designed to move through the full range of motion
 - The appendages are too short for the full range of motion.

In 10/27/16 meeting, it was noted that @arouinfar and I will be working on this.

**Comment by jessegreenberg** [comment]
One possible change we discussed was to switch which leg is in front.

**Comment by jessegreenberg** [comment]
@arouinfar and I met to discuss this.  @arouinfar suggested working on the artwork so that the front leg is draggable and the back leg is planted.  The body will also be shifted down a bit so the new legs are more firmly planted on the rug.

**Comment by arouinfar** [comment]
@jessegreenberg I've separated the artwork for the front leg into two pieces, but I still need to figure out how to tweak them to avoid gaps in the joint while rotating.

<img width="353" alt="screen shot 2016-10-31 at 2 43 08 pm" src="https://cloud.githubusercontent.com/assets/8419308/19870957/7b2cd9ce-9f78-11e6-9acf-55779191160e.png">
<img width="304" alt="screen shot 2016-10-31 at 2 44 22 pm" src="https://cloud.githubusercontent.com/assets/8419308/19870986/a11214ba-9f78-11e6-9591-23199f793196.png">
<img width="368" alt="screen shot 2016-10-31 at 2 44 55 pm" src="https://cloud.githubusercontent.com/assets/8419308/19870992/a7311abc-9f78-11e6-9f5d-92b75107a1e4.png">
<img width="355" alt="screen shot 2016-10-31 at 2 44 38 pm" src="https://cloud.githubusercontent.com/assets/8419308/19870993/a73d80e0-9f78-11e6-8cb5-c0c4707cc184.png">

**Comment by arouinfar** [comment]
@jessegreenberg @ariel-phet here's what I was able to put together.
<img src="https://cloud.githubusercontent.com/assets/8419308/19901216/fb2679f6-a02b-11e6-9768-9f34f883709c.png" width=222><img src="https://cloud.githubusercontent.com/assets/8419308/19901214/fb205c88-a02b-11e6-99c8-77f71440dfdd.png" width=222><img src="https://cloud.githubusercontent.com/assets/8419308/19901215/fb208046-a02b-11e6-83d3-9757435b9f33.png" width=222>

The hyperextended knee doesn't look so great, but I think it's passable.
<img src="https://cloud.githubusercontent.com/assets/8419308/19901216/fb2679f6-a02b-11e6-9768-9f34f883709c.png" width=222><img src="https://cloud.githubusercontent.com/assets/8419308/19901268/2d6dff42-a02c-11e6-9f83-739e646b7f45.png" width=222><img src="https://cloud.githubusercontent.com/assets/8419308/19901267/2d633ba2-a02c-11e6-835a-739bc62d6695.png" width=222>

**Comment by jessegreenberg** [comment]
Looking great @arouinfar, this should be a big improvement and make it easier to layer the charges.  For the hyperextended knee, we could limit the angle so it doesn't go that far.  @jobara also suggested the possibility of rotating from the hip instead of the knee.  Do you think this would help or be doable from the current illustrator artwork?

**Comment by arouinfar** [comment]
@jessegreenberg perhaps this could be the maximum allowable extension
<img src="https://cloud.githubusercontent.com/assets/8419308/19906210/7a77e1b4-a03f-11e6-8008-be3d31d6869d.png" width=350>

If we hinge at the hip instead of the knee, we'd have more mobility in front of the body, but the foot wouldn't be able to go very far behind the the body without things looking pretty weird. Try keeping your torso upright while moving a straight leg behind you. It's not very easy, but perhaps Pulp Fiction John is as flexible as Saturday Night Fever John?  

What would you suggest @ariel-phet?

**Comment by jessegreenberg** [comment]
>  but the foot wouldn't be able to go very far behind the the body without things looking pretty weird.

True!  But if the hip is behind the jacket, I am wondering if it will look less strange than the extended knee.

Regarding limiting the angle, I forgot that there might be value in allowing the foot to completely leave the rug. If the user can't pick up any charges in the air, it would reinforce that charges are coming from the rug.

**Comment by arouinfar** [comment]
@jessegreenberg here's a version with the hip hinge. I think the jacket does a huge favor by hiding the actual hinge. I think I prefer this to the hinging knee.

<img src="https://cloud.githubusercontent.com/assets/8419308/19909630/2baeccf0-a04e-11e6-85cc-c24a363ac55c.png" height=300><img src="https://cloud.githubusercontent.com/assets/8419308/19909632/2bb10808-a04e-11e6-9e56-7311c6863c87.png" height=300><img src="https://cloud.githubusercontent.com/assets/8419308/19909631/2baf0f62-a04e-11e6-8aaa-871d0ed92f31.png" height=300><img src="https://cloud.githubusercontent.com/assets/8419308/19909633/2bb1ec8c-a04e-11e6-95b0-d9e2e8446a1e.png" height=300><img src="https://cloud.githubusercontent.com/assets/8419308/19909629/2bae3222-a04e-11e6-9ef0-f97369f58517.png" height=300>

**Comment by jessegreenberg** [comment]
@arouinfar I agree, I think that looks great.

**Comment by jobara** [comment]
@arouinfar This does look really good. You've captured what appears to be a realistic kicking motion. I can even imagine him kicking a ball down a field in those poses.

**Comment by ariel-phet** [comment]
@arouinfar I thing the full leg swing looks good. I would consider the following:

- tweak the left leg a bit so he is in a more "balanced" position when the right leg is swung fully forward (so basically bring the leg more under his body) Not a huge deal, but I think that would look a bit better

- agreed that the jacket hiding the hinge is a good thing

- some fair warning - we may get sent back to the drawing board on this one, a certain director of the project does not necessarily like the association of violence this image evokes when connected to Pulp Fiction...

**Comment by arouinfar** [comment]
@jobara Thanks! The hip hinge was a great suggestion.

@ariel-phet I agree that straightening the back leg would look more balanced. However, I'm not particularly keen on spending much more time on the artwork if this rendition of John is going to get the axe. 

@jessegreenberg and I took a look at disco-John earlier this week. His 3/4 pose is far more challenging to add a reasonable-looking pivoting leg to. The jacket doesn't hide the hip joint, so it won't look as clean.

Is this something we can briefly discuss during @jessegreenberg's status report?

**Comment by ariel-phet** [comment]
@arouinfar I think would be fine to bring up in status yes

**Comment by arouinfar** [comment]
Next steps
* Give John a haircut
* Send @kathy-phet screenshots of both options
* KP will reach out to the lawyers and BrainPOP before artwork is further edited

**Comment by arouinfar** [comment]
Here's the haircut!
<img width="539" alt="screen shot 2016-11-03 at 10 58 19 am" src="https://cloud.githubusercontent.com/assets/8419308/19976219/7c09a2bc-a1b4-11e6-9d69-bcf54625ec53.png">

**Comment by arouinfar** [comment]
@kathy-phet I've emailed you the image files of John with and without the bun. Please reassign this issue back to me if/when we have the green light to continue down this path.

**Comment by arouinfar** [comment]
Well, yes and no. The artwork was updated (see https://github.com/phetsims/tasks/issues/309), but I believe there was some concern that Travoltage was possibly too photo-realistic, so we explored the cartoonish Travoltage in this issue. There were probably also some issues with CU Legal, which is why this was tagged with `project:requires-kathy`. My best guess is that we can close this issue, but I'll let @ariel-phet make that call.
